web/gds-admin-ui/src/pages/account/Login.js

```diff
@@ -7,19 +7,50 @@ import AccountLayout from './AccountLayout';
 import SignWithGoogle from '../../components/Auth/SignWithGoogle';
 import { loginUser } from '../../redux/auth/actions';
 import config from "../../config";
+import { APICore, setCookie } from '../../helpers/api/apiCore';
+import { getCookie } from '../../utils';
 
+const api = new APICore()
 
 const Login = (): React$Element<any> => {
     const dispatch = useDispatch();
     const { userIsLoggedIn, user } = useSelector((state) => ({
         userIsLoggedIn: state.Auth.userIsLoggedIn,
         user: state.Auth.user
     }))
+    const [authenticated, setAuthenticated] = React.useState(false)
+    const isMounted = React.useRef(true)
```

Comment:
Could you explain what `isMounted` is doing here please? E.g. why is this `useRef` instead of `useState(false)`?

Reply:
`isMounted` is a pattern in React to avoid `memory leaks` due to side effects like data fetching, setting up a subscription, and manually changing the DOM React components.
`useRef` unlike local variables that `useState` takes, React makes sure same reference is returned during each render

Reply:
> @elysee15 this looks very good, thank you for your hard work on this PR. I had a couple of clarifying questions if you wouldn't mind - just so I understand the login workflow completely.
> 
> One other question - where does the profile picture and name get extracted for the header bar? Is that still extracted from the Google credential or is it from the access token?

I take all this from the access token you give me after authentication